src/Canvas.ts

```diff
@@ -515,6 +515,31 @@ export class CanvasForm extends VisualForm {
     
     
     
+    /**
+    * Activate dashed stroke and set dash style. You can customize the segments and offset.
+    * @example `form.dash()`, `form.dash([5, 10])`, `form.dash([5, 5], 5)`
+    * @param segments Dash segments. Defaults to `[5, 5]`. (See [canvas documentation](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/setLineDash))
+    * @param offset Dash offset. Defaults to 0. (See [canvas documentation]()https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/lineDashOffset)
+    */
+    dash( segments:number[]=[5, 5], offset:number=0 ):this {

```

Comment:
instead of `number[]` type, you can use `PtLike` which will accept both `Pt` and `number[]`

Reply:
sure, will change